pyteal/ast/subroutine.py

```diff
@@ -624,18 +690,10 @@ def method_spec(self) -> sdk_abi.Method:
         spec = {
             "name": self.name(),
             "args": args,
-            "returns": {"type": str(self.type_of())},
+            "desc": desc,
```

Comment:
If no description is provided, I'd prefer this field is omitted from the JSON object. I haven't tested, but the code here suggests it will be populated as an empty string.